zuul.d/pipelines.yaml

```diff
@@ -128,7 +128,6 @@
 
 - pipeline:
     name: periodic
-    post-review: true
```

Comment:
FWIW: This should be safe to run, we do it in openstack-infra. It just means projects need to be careful to audit the code they land as to not echo secrets.  Below is a different story as they can be run using recheck